src/build/GhosttyResources.zig

```diff
@@ -303,3 +363,36 @@ pub fn install(self: *const GhosttyResources) void {
     const b = self.steps[0].owner;
     for (self.steps) |step| b.getInstallStep().dependOn(step);
 }
+
+pub fn formatService(b: *std.Build, cfg: *const Config, src: std.Build.LazyPath, dest: []const u8) *std.Build.Step {
+    var cmd = b.addSystemCommand(&.{"sed"});
```

Comment:
Be curious if this works on Mac (GNU vs BSD sed). I'd prefer to just read the file and do the string replacement myself in Zig, honestly, they're small files and the replacement text is easy.

I'm also curious if the Zig build system has something we can use like the C header formatting to do this for us.

Reply:
macOS isn't much of a problem (the templated files are never generated on anything but Linux) but FreeBSD (or other BSDs) might be since they'll need the desktop files as well. 

I just wrote a stupidly simple Zig program to do the templating so the GNU vs BSD sec issue is moot now.